docs/platforms/javascript/guides/nuxt/install/index.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Installation Methods
+sidebar_order: 1.5
+description: "Review our alternate installation methods."
+---
+
+<PageGrid />
+
+Nuxt uses ES Modules for server-side builds, which requires Sentry to register Node [customization hooks](https://nodejs.org/api/module.html#customization-hooks).
+Those customization hooks need to be registered before the rest of the application.
+
+To be able to run Sentry before the rest of the application and fully monitor the server-side, Sentry can be initialized using one of those two approaches:
+
+- Dynamically import server code after initializing Sentry (default)
+- Preload Sentry configuration with the Node `--import` CLI flag
```

Comment:
l: wdyt about this? Otherwise we have pretty much duplicated content on this page.

```suggestion
Nuxt uses ES Modules for server-side builds, which requires Sentry to register Node [customization hooks](https://nodejs.org/api/module.html#customization-hooks).
Those customization hooks need to be registered before the rest of the application.

To be able to run Sentry before the rest of the application and fully monitor the server-side, Sentry can be initialized using one of those two approaches:

<PageGrid />
```